### PR TITLE
MAUI-4752 Getting started UG for Shimmer in .NET MAUI

### PR DESCRIPTION
**Pull request by Priyadharshini-Periyasamy**
### **Description**

- I have added the Getting started UG content in .NET MAUI

### CODE REVIEW

File: MAUI/Shimmer/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        this.Content = shimmer;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Register the handler
+
+The `Syncfusion.Maui.Core` NuGet is a dependent package for all Syncfusion controls of .NET MAUI. In the `MauiProgram.cs` file, register the handler for Syncfusion core.
```
Comment (florenceliliana) [inline MAUI/Shimmer/getting-started.md:55]: The `Syncfusion.Maui.Core` NuGet is a dependent package for all Syncfusion controls of .NET MAUI. In the `MauiProgram.cs` file, register the handler for Syncfusion core.

File: MAUI/Shimmer/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+# Getting Started of .NET MAUI Shimmer (SfShimmer)
+
+This section explains how to add the Shimmer control.
+
+## Creating an application using the .NET MAUI Shimmer
+
+1. Create a new .NET MAUI application in Visual Studio.
+
+2. Syncfusion .NET MAUI components are available on [nuget.org](https://www.nuget.org/). To add `SfShimmer` to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Maui.Core](https://www.nuget.org/packages/Syncfusion.Maui.Core), and then install it.
```
Comment (florenceliliana) [inline MAUI/Shimmer/getting-started.md:18]: 2. Syncfusion .NET MAUI components are available on [nuget.org](https://www.nuget.org/). To add `SfShimmer` to your project, open the NuGet package manager in Visual Studio, and search for the [Syncfusion.Maui.Core](https://www.nuget.org/packages/Syncfusion.Maui.Core), and then install it.

**Review by florenceliliana — CHANGES_REQUESTED** [review]

File: MAUI/Shimmer/getting-started.md (hunk 2 above)
Comment (Priyadharshini-Periyasamy) [reply MAUI/Shimmer/getting-started.md]: Comments addressed

File: MAUI/Shimmer/getting-started.md (hunk 1 above)
Comment (Priyadharshini-Periyasamy) [reply MAUI/Shimmer/getting-started.md:54]: Comments modified

File: MAUI/Shimmer/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+        }
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Change different Shimmer views
+
+The `.NET MAUI Shimmer` control provides seven different Shimmer types of views, it can be assigned to the control using the `Type` property. By default, the control is assigned to the `CirclePersona` view.
```
[the remark this answers is not in the document]
Comment (Priyadharshini-Periyasamy) [reply MAUI/Shimmer/getting-started.md]: Contents modified

**Review by Sasikumar3595 — APPROVED** [review]
Code changes are fine

**Review by DeepakRajSundar — APPROVED** [review]

**Pull request closed without merging**